changes.go

```diff
@@ -240,7 +248,9 @@ type AddMachineParams struct {
 	// this machine is a container, in which case also ContainerType is set.
 	ParentId string
 
+	existing           bool
 	bundleMachineID    string
+	machineID          string
```

Comment:
Comment explaining difference between `bundleMachineID`, `machineID` and `containerMachineID`?